src/api/routes/user.ts

```diff
@@ -55,11 +58,20 @@ export default () => {
       const { id } = req.params;
       const command = req.body as UpdateUserCommand;
 
-      if (
-        !accessManagerFactory
-          .forAuthentication(getAuthenticationOrFail(req))
-          .isLoggedInAsUser(new UserId(id))
-      ) {
+      const userId = new UserId(id);
+
+      const isLoggedInAsUser = accessManagerFactory
+        .forAuthentication(getAuthenticationOrFail(req))
+        .isLoggedInAsUser(userId);
+
+      const hasAccessPassForUser = await accessManagerFactory
+        .forAuthentication(getAuthenticationOrFail(req))
+        .hasAccessPassForUser(userId);
+
+      if (!isLoggedInAsUser) {
+        if (hasAccessPassForUser) {
+          throw new ApiError(405, apiErrorCodes.ACCESS_DENIED);
```

Comment:
Shouldn't Access denied be `403`?  Method allowed tends to mean that this path does not support this HTTP verb.

Reply:
I already changed this, just hadn't pushed.  One sec...